app/templates/groups/edit/followers.hbs

```diff
@@ -0,0 +1,16 @@
+<GroupNav @group={{this.model.group}} />
```

Comment:
@maze-runnar After adding the navigation, if you try to save the event settings or events to the group it throws an error.

Reply:
Oh, I will check and try to fix it. If not possible then will revert.